src/tests/ftest/io/parallel_io.py

```diff
@@ -102,6 +123,42 @@ def start_dfuse(self):
                            exc_info=error)
             self.fail("Test was expected to pass but it failed.\n")
 
+    def stat_bfree(self, path):
+        """Get stat bfree
+
+          Args:
+            path (str): path to get free block size of.
+          Returns:
+            integer value of stat free blocks
+        """
+        cmd = "ssh" " {}@{}" " stat -c%a -f {}".format(
+            getuser(), self.hostlist_clients[0], path)
+        try:
+            result = subprocess.check_output(cmd, shell=True)
```

Comment:
subprocess call with shell=True will trigger a high-severity security issue (will be caught by bandit).
Please provide a "list" for cmd, and remove the "shell=True".
Please see lanuch.py as reference.